model/risk_model.py

```
@@ -1,7 +1,810 @@
+        """
+        3개 주소 동시 검증
+        
+        Args:
+            property_addr: 매물 등록 주소
+            registry_addr: 등기부등본 주소  
+            building_addr: 건축물대장 주소
+            
+        Returns:
+            bool: 주소 일치 여부
+        """
+        logger.info(f"주소 검증 시작:")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)